packages/link/src/spectrum-link.css

```diff
@@ -39,18 +39,19 @@ a:active {
     --mod-link-text-color: var(--mod-link-text-color-active, var(--mod-link-text-color-primary-active, var(--spectrum-accent-content-color-down)));
 }
 
-a:focus-visible {
+:host([quiet]) a {
+    -webkit-text-decoration: none;
+    text-decoration: none;
+}
+
+:host a:focus-visible,
```

Comment:
Is the :host selector included here for specificity? If it's not needed, I do think it reads cleaner without it.